cfgov/unprocessed/css/on-demand/secondary-navigation.less

```diff
@@ -24,10 +24,10 @@
 @import (reference) 'cf-typography.less';
 
 // The Capital Framework (CF) brand color palette.
-@import (reference) "../brand-palette.less";
+// @import (reference) "../brand-palette.less";
```

Comment:
Can we delete this commented out bits or add a `TODO` on when they can be deleted?

Reply:
oops, meant to delete it - thanks!